DQMServices/Core/src/DQMStore.cc

```diff
@@ -856,7 +707,7 @@ DQMStore::book_(const std::string &dir, const std::string &name,
   {
     // Create and initialise core object.
     assert(dirs_.count(dir));
-    MonitorElement proto(&*dirs_.find(dir), name, run_, streamId_, moduleId_);
+    MonitorElement proto(&*dirs_.find(dir), name, run_, moduleId_);
```

Comment:
@fwyzard ugh... What have you done there? Did you also remove the `streamId` from the `MonitorElement` constructor? It should be removed, but the change is not in this PR, so now `streamId`s and `moduleId`s are messed up!